packages/clients/src/scw/locality.ts

```diff
@@ -8,4 +9,5 @@ export type Zone =
   | 'nl-ams-2'
   | 'pl-waw-1'
   | 'pl-waw-2'
-  | string
+  | (string & {})
+/* eslint-enable @typescript-eslint/ban-types */
```

Comment:
I think we can safely remove this `eslint-enable`, `eslint-disable` is only valid for the current file

Reply:
You mean removing the disable at the end?

Because:

> Requires 'eslint-enable' directive for '@typescript-eslint/ban-types'.eslint[eslint-comments/disable-enable-pair](https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/disable-enable-pair.html)